src/engine/SCons/Platform/win32.py

```diff
@@ -178,61 +178,68 @@ def piped_spawn(sh, escape, cmd, args, env, stdout, stderr):
     #   we redirect it into a temporary file tmpFileStdout
     #   (tmpFileStderr) and copy the contents of this file
     #   to stdout (stderr) given in the argument
+    # Note that because this will paste shell redirection syntax
+    # into the cmdline, we have to call a shell to run the command,
+    # even though that's a bit of a performance hit.
     if not sh:
         sys.stderr.write("scons: Could not find command interpreter, is it in your PATH?\n")
         return 127
-    else:
```

Comment:
Review note: sorry about this one, the `else` clause didn't need to be an else-indented block since the `if` block returns unconditionally. Actual non-whitespace changes are to lines 186/187 and the `*Redirected` flags explicity becoming True/False; `mkstemp` returns a tuple instead of `mktemp` returning a filename.